decred/decred/dcr/dcrdata.py

```diff
@@ -687,12 +685,11 @@ def blockHeader(self, hexHash):
             BlockHeader: An object which implements the BlockHeader API.
         """
         try:
-            serialized = self.headerDB[hashFromHex(hexHash).bytes()]
-            return msgblock.BlockHeader.deserialize(serialized)
+            return self.headerDB[hashFromHex(hexHash).bytes()]
```

Comment:
The returned object is an already deserialized `BlockHeader`, because of the Blobber interface associated to the `headerDB` attribute.